appserver/jdbc/jdbc-runtime/src/main/java/org/glassfish/jdbcruntime/deployment/annotation/handlers/DataSourceDefinitionHandler.java

```diff
@@ -368,21 +369,28 @@ private void merge(Set<ResourceDescriptor> dsdDescs, DataSourceDefinition defn)
 
     }
 
+    /**
```

Comment:
Why introduce this method? You can use static import just the same

Reply:
fixed